test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -208,3 +208,55 @@ TEST_F(YacGateTest, LoadBlockWhenDifferentCommit) {
 
   ASSERT_TRUE(gate_wrapper.validate());
 }
+/**
+ * @given yac gate
+ * @when recives new commit different to the one it voted for
+ * @then polls nodes for the block with corresponding hash untill it succeed
```

Comment:
This description also applies to the previous test. It should be stated that it does not succeed with download on the first try.

Reply:
untill -> until